config/locales/en.yml

```diff
@@ -2592,7 +2592,7 @@ en:
   label_none_parentheses: "(none)"
   label_not_contains: "doesn't contain"
   label_not_equals: "is not"
-  label_life_cycle_plural: "Project lifecycle"
+  label_life_cycle_step_plural: "Project lifecycle"
```

Comment:
```suggestion
  label_life_cycle_steps: "Project lifecycle"
```

Reply:
I changed that explicitly after noticing that a translation was no longer applied. I currently don't remember which component but there is a hard wired expectation that there is a label with a plural suffix. Otherwise I wouldn't even have put it into the way to nondescript label section.